backend/script/load_teams.py

```diff
@@ -5,6 +5,8 @@
 from ..models import TeamData
 from ..db import engine
 import polars as pl
+import argparse
```

Comment:
Is argparse included in the default distribution of python? If not can you put it in the requirements.txt?

Reply:
argparse is included in default distribution!